plugins/API/WidgetMetadata.php

```diff
@@ -178,7 +178,7 @@ private function buildCategoryMetadata($category)
 
         return array(
             'id'    => (string) $category->getId(),
-            'name'  => Piwik::translate($category->getId()),
+            'name'  => Piwik::translate($category->getDisplayName()),
```

Comment:
FYI the Piwik::translate is not needed here

Reply:
whoops